commit 40a219753cea7b5ccf362288f4ee46fdfba87c72
Author: Les Aker <me@lesaker.org>
Date:   Mon Sep 12 21:37:05 2016 -0400

    handle certs with no org or CN field

diff --git a/cmd/format/format.go b/cmd/format/format.go
index c3bbc51..bf39d26 100644
--- a/cmd/format/format.go
+++ b/cmd/format/format.go
@@ -65,6 +65,10 @@ func applyNamingConv(file []byte, filename string) error {
 	name := ""
 	// If CommonName field is empty, then use the Organization name instead
 	if len(response.Subject.CommonName) == 0 {
+		if len(response.Subject.Organization) == 0 {
+			fmt.Printf("Failing to rename due to missing CN and Org fields: %s", filename)
+			return nil
+		}
 		name = strings.Replace(response.Subject.Organization[0], " ", "", -1)
 		name = strings.Replace(name, "/", "-", -1)
 	} else {
